### PR TITLE
Add menu seeder

### PR DESCRIPTION
**Pull request by thaoratngau**
Add menu seeder

### CODE REVIEW

**Comment by thaoratngau** [comment]
ready

**Comment by thaoratngau** [comment]
ready

File: database/seeds/MenuTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public function run()
+    {
+        $count = 0;
+        foreach ($this->records as $key => $record){
+            try{
+                $record['created_at'] = new DateTime();
+                $record['updated_at'] = new DateTime();
+                \Illuminate\Support\Facades\DB::table('menu_items')->insert($record);
+                $count++;
+            }catch (Exception $e){
```
Comment (BunnyPi04) [inline database/seeds/MenuTableSeeder.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by thaoratngau** [comment]
ready

**Comment by BunnyPi04** [comment]
commented

**Comment by thaoratngau** [comment]
ready

**Pull request merged**